### DISCUSSION TITLE
Can we store the job variable in mongoDb?

### DISCUSSION THREAD

**Discussion by ObaidAshiq** [Hexagon/croner#235] [Q&A]
I want to store job variable in mongoDb because right now ,whenever i restart the server my variable (job) loses its configuration?

**Comment by Hexagon** [comment]
It's not supported at the moment, but it would be possible to make this easier in a future release. I'll keep this in mind :+1:

**Comment by ObaidAshiq** [comment]
If your try to do this for this version, it throw an error
Error inserting document: BSONError: Cannot convert circular structure to BSON

![image](https://github.com/Hexagon/croner/assets/133643704/ac12cddd-16b6-4d84-ae12-641f2f81937e)
Any ETA for this feature
And any workarounds for now ?
@Hexagon
